Source/Scene/PostProcessSampleMode.js

```diff
@@ -0,0 +1,10 @@
+define(function() {
+    'use strict';
+
+    var PostProcessSampleMode = {
```

Comment:
Should this be explicitly `@private`?

Reply:
No, this is an option to `PostProcess` for how to sample the input texture. Most will use nearest but our implementation of blur and FXAA need linear sampling as part of the optimized versions.